kuzzle-esp32/01-rgb-led/rgb-led.md

```diff
@@ -0,0 +1,333 @@
+# USE KUZZLE AS AN IOT BACKEND WITH ESP32
+
+![kuzzle](img/lanceur-backend-1900.jpg)
+
+## Intro: Use Kuzzle as an IoT Backend With ESP32
+
+Kuzzle is an open-source backend suite that can be installed on-premises and includes a multi-protocol API that can be used to integrate IoT devices.
+
+In this article we will show you how to develop an IoT application using an ESP32 module and Kuzzle Backend, communicating using MQTT.
+
+Specifically, we will build an RDB LED that we can control from the Kuzzle Console to change its color.
+
+## Installing Kuzzle Server and Kuzzle Admin Console
+
+First you will need to install Kuzzle Backend, an open source IoT backend solution, on a your favorite Linux distribution. You can follow this Getting Started guide to get Kuzzle Backend up and running <http://docs.kuzzle.io/guide/getting-started/>.
+
+Next you’ll have to install and enable the Kuzzle Backend mqtt-protocol plugin. This is a plugin that allows Kuzzle Backend to communicate using the MQTT protocol. It can be downloaded from <https://github.com/kuzzleio/protocol-mqtt.> Instructions on installing the protocol can be found in the Install section.
+
+Then you can install Kuzzle Console or use the online version. This is the administration tool that allows you to manage and browse your data.The online version of Kuzzle Console can be configured to connect to your local Kuzzle Server (although it is hosted online, all the code to access your Kuzzle Backend instance will run locally in your browser, thus there is no need for NAT).
```

Comment:
typo: `Kuzzle Admin Console` and maybe you can put an hyperlink to the online version here ?

Reply:
Actually `Kuzzle Console` is not a type as during the phase where we where branding our products, we opted for Kuzzle Console